Resources/Prototypes/Catalog/uplink_catalog.yml

```diff
@@ -172,6 +172,17 @@
   categories:
   - UplinkWeaponry
 
+- type: listing
+  id: UplinkRadPen
+  name: uplink-radpen-name
+  description: uplink-radpen-desc
+  icon: { sprite: /Textures/Objects/Misc/pens.rsi/, state: pen }
```

Comment:
state should match the meta.json name "radpen"

Reply:
It shows the normal pen for the exploding and hypopen, too. So this one is correct as it is.

Reply:
guhhhh what's the yaml error then